presidio-analyzer/presidio_analyzer/recognizer_registry/recognizer_registry_provider.py

```diff
@@ -255,9 +255,19 @@ def create_recognizer_registry(self) -> RecognizerRegistry:
             fields[field] = self.configuration.get(field, self.default_values[field])
 
         self.supported_languages = fields["supported_languages"]
+        self.global_regex_flags = fields["global_regex_flags"]
```

Comment:
Add to init

Reply:
That's what I did initially, just take note that you still have to do this assignment because the functions in there will still `self.supported_lauguages` etc